codice/ddf
# DDF-4494 Migrate theme-settings to react

#### What does this PR do?
Refactors theme-settings from marionette to react 
#### Who is reviewing it? 
@Schachte @zta6 @adimka @gordocanchola 
#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining
@djblue

#### How should this be tested?
Test each field in theme setting by changing the values see the display update, then reload the UI in a new browser and verify the changes are visible (persisted in user settings)
#### Any background context you want to provide?
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4494 ____

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
